clients/src/main/java/org/apache/kafka/clients/NetworkClient.java

```diff
@@ -1222,6 +1249,95 @@ private InProgressData(int requestVersion, boolean isPartialUpdate) {
 
     }
 
+    class TelemetrySender {
+
+        private final ClientTelemetrySender clientTelemetrySender;
+        private Node stickyNode;
+
+        public TelemetrySender(ClientTelemetrySender clientTelemetrySender) {
+            this.clientTelemetrySender = clientTelemetrySender;
+        }
+
+        public long maybeUpdate(long now) {
+            long timeToNextUpdate = clientTelemetrySender.timeToNextUpdate(defaultRequestTimeoutMs);
```

Comment:
to be more consistent, can we do `long nowMs` and `long timeToNextUpdateMs = ...`

Reply:
I have tried to be consistent with the other methods in `NetworkClient` where metadata has `long timeToNextMetadataUpdate = metadata.timeToNextUpdate(now);`, also close and disconnect has `long now = time.milliseconds();`.

I have made the changes in other PR to include `ms` to be consistent as the file was newly introduced but seems it's mixed in clients code and trying to be consistent (best efforts) for the respective files. Please let me know if you disagree and I should include `ms` anyways.